doc/sphinx/source/community/backward_compatibility.rst

```diff
@@ -0,0 +1,262 @@
+===========================================
+
+V1.0. Agreed with ESMValTool community during October 2022 workshop.
+
+Motivation
+----------
+
+Development of recipes or conducting project-related work may require a
+rather long period of time during which new versions of the ESMValTool
+might become available. For a good user experience and a seamless
+workflow, users and developers need to know before upgrading to a new
+version if and how their work might be affected (backward
+compatibility). This includes, for instance, information about changes
+to technical features such as syntax of recipes and configuration files,
+and interfaces of shared functions, but also changes that affect the
+results of an ESMValTool run, e.g. modification of algorithms or changes
+to the order of operators. It is therefore essential that users and
+developers have the best advice on how and when to upgrade to new
+versions.
+
+While trying to minimise the impact of new code developments on users
+and developers by maintaining backward compatibility where possible,
+this cannot always be guaranteed. A very restrictive policy might delay
+the ESMValTool development and make it more complex for developers to
+contribute.
+
+This document outlines the key principles and proposals of an updated
+ESMValTool policy on backward compatbility.
+
+Definitions
+-----------
+
+**Release:** A numbered version of ESMValCore / ESMValTool that has been
+released to the community, e.g. 2.4.0. This policy relates only to
+backward compatibility of releases, not to interim revisions of the main
+branch. Release numbers are of the format x.y.z, where:
+
+-  x indicates a major release
+-  y indicates a minor release
+-  z indicates a patch release
+
+**Backward-incompatible change:** An ESMValTool change that causes a
+recipe to no longer run successfully (a *breaking change*), or which
+results in scientifically significant changes in results (a *science
+change*).
+
+**Breaking change:** A change which causes a previously working recipe
+to no longer run successfully.
+
+**Science change:** A change that alters scientific results. We do not
+formally distinguish between trivial science changes (e.g. from changes
+in the order of calculations) and more significant changes that would
+affect interpretation, although the detail that we communicate will
+share any understanding that we have regarding expected impact.
+
+**Benign third-party dependency changes:** A change over which we have
+no control, but which we believe will only have trivial technical
+impacts (such as a change in font). Such changes are outside of the
+scope of this policy, though we will communicate about those we are
+aware of.
+
+**Developer of backward-incompatible change:** For the purpose of this
+policy, developer is the individual that is responsible for the pull
+request (PR) that is not backward compatible. In the vast majority of
+cases this will be an ESMValCore development.
```

Comment:
```suggestion
request (PR) that is not backward compatible.
```
How do we know that? And does it make the document more clear to add that sentence?